app/assets/javascripts/admin/components/theme-setting-relatives-selector.js.es6

```diff
@@ -0,0 +1,33 @@
+import Component from "@ember/component";
+import BufferedContent from "discourse/mixins/buffered-content";
+import SettingComponent from "admin/mixins/setting-component";
+import { ajax } from "discourse/lib/ajax";
+
+export default Component.extend(BufferedContent, SettingComponent, {
+  layoutName: "admin/templates/components/site-setting",
+
+  _save() {
+    return ajax(`/admin/themes/${this.model.id}`, {
+      type: "PUT",
+      data: {
+        theme: {
+          [this.setting.setting]: this.convertNamesToIds()
+        }
+      }
+    }).then(() => {
+      this.store.findAll("theme");
+    });
+  },
+
+  convertNamesToIds() {
+    return this.get("buffered.value")
+      .split("|")
+      .map(theme_name => {
+        if (theme_name !== "") {
+          return this.setting.allThemes.find(theme => theme.name === theme_name)
```

Comment:
This could be inefficient if there are many names and themes. In practice we might never see it but it's worth pointing out.

Reply:
@eviltrout thank you for your review. I am thinking about that point. Do you think it would be better to send a string to backend and backend will figure out ids?
This is currently happening for example to reserved username setting
![Screenshot from 2019-11-27 09-32-51](https://user-images.githubusercontent.com/72780/69678230-12691880-10f9-11ea-827a-fe03acd6886a.png)